gtsofa/portfolio
# Add responsiveness to the website

### What is this PR supposed to do?
- Add breakpoints for large tablets and desktop devices
- Add breakpoints for smaller devices such as mobile phones
### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out this branch
- Open the website on a browser and test on different screen sizes
### Relevant PT stories
- None
